pytorch_lightning/strategies/tpu_spawn.py

```diff
@@ -148,8 +141,7 @@ def distributed_sampler_kwargs(self) -> Dict[str, int]:
 
     @property
     def is_distributed(self) -> bool:
-        # HOST_WORLD_SIZE is None outside the xmp.spawn process
-        return os.getenv(xenv.HOST_WORLD_SIZE, None) and self.world_size != 1
+        return self.world_size > 1
```

Comment:
@awaelchli I think I added this comment to support Pod training. Need to verify.